src/renderer/reader/components/Reader.tsx

```diff
@@ -605,7 +605,11 @@ class Reader extends React.Component<IProps, IState> {
     }
 
     private async handleReadingLocationChange(loc: LocatorExtended) {
+
+        // Why findBookmark here ? can be removed
```

Comment:
I agree, but we have to make sure that `checkBookmarks()` is called (automatically from `componentDidUpdate()` if `oldState.bookmarks !== this.state.bookmarks`).